main.go

```diff
@@ -238,6 +241,14 @@ func run(state overseer.State) {
 		if err = e.ScanFileSystem(ctx, sources.NewConfig(fs)); err != nil {
 			logrus.WithError(err).Fatal("Failed to scan filesystem")
 		}
+	case fileScan.FullCommand():
+		fs := func(c *sources.Config) {
+			c.FilePath = *fileScanPath
+		}
```

Comment:
**Question:** Does this mean we will only scan one file at a time?

Reply:
Yep, I was adding the same functionality as enterprise. Ideally we use only one implementation and then that can handle multiple files, but for now I felt for MVP it was sufficient.